src/app/pages/subjects/subjects.js

```diff
@@ -10,29 +9,17 @@ function useFeatureFlag() {
     return flag;
 }
 
-function SelectedComponent({featureFlag}) {
-    const OldPage = loadable({
-        loader: () => import('./old/subjects.js'),
-        loading: LoadingPlaceholder
-    });
-    const NewPage = loadable({
-        loader: () => import('./new/subjects.js'),
-        loading: LoadingPlaceholder
-    });
-
-    return (
-        featureFlag ? <NewPage /> : <OldPage />
-    );
-}
+const NewPage = loadable({
```

Comment:
how have you guys liked using this `loadable` package? seems cool, hadn't heard about it before